CONTRIBUTING.md

```diff
@@ -113,12 +126,13 @@ $short         = foo($bar);
 $long_variable = foo($baz);
 ```
```

Comment:
I think it's a bad idea to write a code snippet to illustrate a bad exemple.
It's likely that only the code be read, and contributors could think they have to use this syntax